rotkehlchen/tests/unit/decoders/test_polygon_pos_bridge.py

```diff
@@ -0,0 +1,468 @@
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=205,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=Asset('eip155:1/erc721:0xDF74156420Bd57ab387B195ed81EcA36F9fABAca'),
+            balance=Balance(amount=ONE),
+            location_label=user_address,
+            notes='Receive Exit NFT from Polygon Bridge',
+            tx_hash=tx_hash,
+            counterparty=CPT_POLYGON,
+            address=ZERO_ADDRESS,
+            extra_data={
+                'token_id': 588708823241823398655445372856552995073191575662,
+                'token_name': 'ERC721 token',
+            },
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xddeB04d7Dab85BD6AdfaC8851F9cdB3a06Be77aD']])
+def test_polygon_pos_bridge_plasma_process_exit(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0x772c7c665a24954b11aa66226a4602537e970df114b0d7875572975b641ba47f')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    user_address, timestamp, gas_amount, bridge_amount = ethereum_accounts[0], TimestampMS(1730059799000), '0.000750908659746456', '2004.054999809184671214'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=335,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=Asset('eip155:1/erc721:0xDF74156420Bd57ab387B195ed81EcA36F9fABAca'),
+            balance=Balance(amount=ONE),
+            location_label=user_address,
+            notes='Return Exit NFT to Polygon Bridge',
+            tx_hash=tx_hash,
+            counterparty=CPT_POLYGON,
+            address=ZERO_ADDRESS,
+            extra_data={
+                'token_id': 588708823241823398655445372856552995073191575662,
+                'token_name': 'ERC721 token',
+            },
```

Comment:
why store this? does it come from your decoder?

Reply:
No this isn't added by my decoder - I think it comes from here: https://github.com/rotki/rotki/blob/develop/rotkehlchen/chain/evm/decoding/base.py#L194